examples/complete/variables.tf

```diff
@@ -16,14 +16,32 @@ variable "resource_tags" {
   default     = []
 }
 
+# region needs to provide cross region support.
 variable "region" {
   description = "Region where resources will be created"
   type        = string
-  default     = "eu-gb"
+  default     = "us-south"
+}
+
+variable "cross_region_location" {
+  description = "Specify the cross-regional bucket location. Supported values are 'us', 'eu', and 'ap'."
+  type        = string
+  default     = "us"
+
+  validation {
+    condition     = var.cross_region_location == null || can(regex("us|eu|ap", var.cross_region_location))
+    error_message = "Variable 'cross_region_location' must be 'us' or 'eu', 'ap', or 'null'."
```

Comment:
for the sake of this example, null is not a valid option since we are deploying a cross region bucket